notebooks/post-startup.sh

```diff
@@ -8,9 +8,14 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-# Send stdout and stderr from this script to a tmp file for debugging.
+# The linux user that JupyterLab will be running as. It's important to do some parts of setup in the
+# user space.
+readonly JUPYTER_USER="jupyter"
+
 cd /tmp || exit
-exec >> /tmp/post-startup-output.txt
+# Send stdout and stderr from this script to a file for debugging.
+mkdir /home/"${JUPYTER_USER}"/.terra-debug
+exec >> /home/"${JUPYTER_USER}"/.terra-debug/post-startup-output.txt
```

Comment:
Perhaps you could put this debug output under `$HOME/.terra`. The CLI will put logs in `$HOME/.terra/logs/` and it would be nice to have this consistent. e.g. So we can tell a user with a problem, send me the `$HOME/.terra/....` directory for debugging.